Makefile

```diff
@@ -0,0 +1,46 @@
+# Make settings - @see https://tech.davis-hansson.com/p/make/
```

Comment:
This file was largely taken from the Java SDK with a few tweaks. Hat tip to @agamkrbit for putting this together! 🎉